evalml/tests/automl_tests/test_automl.py

```diff
@@ -1311,6 +1311,8 @@ def test_max_iteration_works_with_stacked_ensemble(mock_pipeline_fit, mock_score
         assert not pipeline_names.str.contains('Ensemble').any()
     elif use_ensembling:
         assert pipeline_names.str.contains('Ensemble').any()
+        ensemble_nth_batch = len(automl.allowed_pipelines) + 1
+        assert f"Ensembling will run every {ensemble_nth_batch} batches."
```

Comment:
I think this is a typo since it always evaluates to True?

Currently, we don't output a message of when ensembling will run if the user specifies `max_iterations` instead of `max_batches`. Maybe we can say "Ensembling will run at the kth iteration and every j iterations after that"?

Reply:
thanks for catching this hehe, updated!